luke/index.js

```diff
@@ -0,0 +1,41 @@
+// task 1
+let snackServings = 26;
+let guests = 9;
+
+// task 2
+console.log(snackServings - guests);
+console.log(guests - snackServings);
+
+// task 3
+console.log(snackServings * guests);
+
+// task 4
+console.log(snackServings / guests);
+
+// task 5
+let drinks = 10;
+let resultOne = (snackServings * guests) / drinks;
+console.log(resultOne);
+
+// task 5.2
+let monkeys = 15;
+let bananas = 9;
+console.log(monkeys / bananas);
+
+// task 6
+let monkeyFights = 20;
+let resultTwo = (monkeys + bananas) * monkeyFights;
+console.log(resultTwo);
+
+// task 7
```

Comment:
Your answers about 8. and 9. are wrong. You can increment and decrement the variables like this :
8.  console.log(++monkeys);
9.  console.log(--bananas);

Reply:
yeah i looked this up today actually and yeah have changed that and pushing now, thanks ciamac!

Reply:
You're welcome:)